neuron/src/lib/Neuron/Zettelkasten/Zettel/Meta.hs

```diff
@@ -45,25 +50,38 @@ instance FromYAML Meta where
 --     mapping
 --       [ "title" .= title,
 --         "tags" .= tags,
---         "date" .= date
+--         "created" .= date
 --       ]
 
-instance FromYAML Day where
+instance FromYAML LocalTime where
   parseYAML =
     parseZettelDate <=< parseYAML @Text
 
-instance ToYAML Day where
+instance ToYAML LocalTime where
   toYAML =
-    toYAML . formatZettelDate
+    toYAML . formatZettelLocalTime
 
--- | The format in which we decode and encode zettel dates.
-zettelDateFormat :: String
-zettelDateFormat = "%Y-%m-%d"
+-- | The default format in which we decode and encode zettel dates.
+defaultZettelDateFormat :: String
+defaultZettelDateFormat = "%Y-%m-%dT%H:%M"
+
+-- | All formats of user-edited tags 'created', that can be parsed.
+zettelDateFormats :: [String]
+zettelDateFormats = ["%Y-%m-%d", defaultZettelDateFormat]
+
+formatZettelLocalTime :: LocalTime -> Text
+formatZettelLocalTime =
+  toText . formatTime defaultTimeLocale defaultZettelDateFormat
 
 formatZettelDate :: Day -> Text
 formatZettelDate =
-  toText . formatTime defaultTimeLocale zettelDateFormat
+  toText . formatTime defaultTimeLocale "%Y-%m-%d"
+
+parseZettelDate :: (MonadFail m, Alternative m) => Text -> m LocalTime
+parseZettelDate t =
+  let s = toString t
+      pars = parseDateM s
+   in asum $ pars <$> zettelDateFormats
 
-parseZettelDate :: MonadFail m => Text -> m Day
-parseZettelDate =
-  parseTimeM False defaultTimeLocale zettelDateFormat . toString
+parseDateM :: MonadFail m => String -> String -> m LocalTime
+parseDateM s = \fmt -> parseTimeM False defaultTimeLocale fmt s
```

Comment:
`parseDateM fmt = ...` and then `asum $ (flip parseDateM $ toString t) <$> zettelDateFormats` above.

Reply:
yep.. I knew there was a 'flip' in it :)